motr/st/utils/motr_rmw_st.sh

```diff
@@ -144,7 +145,7 @@ write_and_update()
 		break
 	}
 	echo "m0cat"
-	$motr_st_util_dir/m0cat -G $MOTR_PARAMS -o $object_id -s $block_size \
+	$motr_st_util_dir/m0cat $di_flag $MOTR_PARAMS -o $object_id -s $block_size \
```

Comment:
This Codacy warning can be skipped/ignored, I think.

Reply:
done/ignored.